src/sentry/demo/demo_org_manager.py

```diff
@@ -52,27 +54,32 @@ def create_demo_org(quick=False) -> Organization:
                 name="Python", organization=org, platform="python"
             )
             python_project.add_team(team)
+            projects.append(python_project)
```

Comment:
@roggenkemper we have a lot of repeetitive code here. Might be nice to make a function within the scope of this function that will do the following:
1. Create a project
2. Add the default team to that project
3. Add that project to the list of projects
4. Return that project

Make sense?

Reply:
so much cleaner . also easier to add new projects now. great idea!